### ISSUE TITLE
Check if our coverage of Royal Colleges is comprehensive, and if not make it so

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/whatdotheyknow-theme#911]
https://www.whatdotheyknow.com/body?tag=royal_college

A list of those recognised by the GMC appears to be at:

https://www.gmc-uk.org/education/reports-and-reviews/annual-specialty-reports

**Comment by RichardTaylor** [comment]
Done. 

I've included faculties of royal colleges and joint bodies made up of royal colleges under the tag royal_college.  The bodies I've added as faculties / joint bodies have their own websites, domain names, governance etc so they appear to be bodies in their own right, and certainly worthy of a separate listing on WhatDoTheyKnow.

Interestingly the [Royal College of Veterinary Surgeons is subject to FOI](https://www.legislation.gov.uk/uksi/2005/3593/schedule/2/made), but the various Royal Colleges of [human] Surgeons are not.
